src/main/java/net/bteuk/network/gui/tutorials/TutorialsGui.java

```diff
@@ -0,0 +1,337 @@
+package net.bteuk.network.gui.tutorials;
+
+import net.bteuk.network.Network;
+import net.bteuk.network.gui.Gui;
+import net.bteuk.network.utils.NetworkUser;
+import net.bteuk.network.utils.SwitchServer;
+import net.bteuk.network.utils.Utils;
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.text.format.NamedTextColor;
+import net.kyori.adventure.text.format.TextDecoration;
+import org.bukkit.Bukkit;
+import org.bukkit.Material;
+import org.bukkit.inventory.ItemStack;
+import teachingtutorials.guis.Event;
+import teachingtutorials.guis.EventType;
+import teachingtutorials.tutorials.Lesson;
+import teachingtutorials.tutorials.Tutorial;
+import teachingtutorials.utils.User;
+
+import static net.bteuk.network.utils.NetworkConfig.CONFIG;
+
+public class TutorialsGui extends Gui {
+
+    private final NetworkUser user;
+
+    public TutorialsGui(NetworkUser user) {
+
+        super(27, Component.text("Tutorials Menu", NamedTextColor.AQUA, TextDecoration.BOLD));
+
+        this.user = user;
+
+        createGui();
+
+    }
+
+    /**
+     * Creates the icons and actions for this menu
+     */
+    private void createGui() {
+
+        //Retrieves the critical information for the menu
+        boolean bCompulsoryTutorialEnabled = CONFIG.getBoolean("tutorials.compulsory_tutorial");
+
+        //Gets the information about the Tutorials User
+        User tutorialsUser = new User(this.user.player);
+        tutorialsUser.fetchDetailsByUUID(Network.getInstance().getTutorialsDBConnection());
+        tutorialsUser.calculateRatings(Network.getInstance().getTutorialsDBConnection());
+
+        //Initiates the current tutorial object
+        Tutorial currentTutorial = new Tutorial();
+
+        //Get compulsory tutorial ID
+        int iCompulsoryTutorialID;
+        Tutorial[] compulsoryTutorials = Tutorial.fetchAll(true, true, Network.getInstance().getTutorialsDBConnection());
+        if (compulsoryTutorials.length == 0)
+            iCompulsoryTutorialID = -1;
+        else
+            iCompulsoryTutorialID = compulsoryTutorials[0].getTutorialID();
```

Comment:
Not that it's wrong, but if you make the if statement this way you could just as well make it a 1-liner.
iCompulsoryTutorialID = compulsoryTutorial.length == 0 ? -1 : compulsoryTutorials[0].getTutorialID();

Reply:
True. I would imagine the compiler would change it to this upon compiling